extensions/interactions/interaction_specs.json

```diff
@@ -409,7 +409,16 @@
     "is_linear": false,
     "display_mode": "inline",
     "id": "SetInput",
-    "customization_arg_specs": [],
+    "customization_arg_specs": [
+      {
+        "description": "Customize the \"Add\" button text:",
```

Comment:
Same note about this needing to be a description. (Note that the fields that might look like instructions in interaction_specs.json are actually boolean fields.)

Reply:
Done!